cdap-master/pom.xml

```diff
@@ -254,6 +365,27 @@
               </execution>
             </executions>
           </plugin>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-antrun-plugin</artifactId>
+            <version>1.7</version>
+            <executions>
+              <execution>
+                <id>rename-plugin-json</id>
+                <phase>process-resources</phase>
+                <goals>
+                  <goal>run</goal>
+                </goals>
+                <configuration>
+                  <target>
+                    <move file="${stage.opt.dir}/artifacts/ETLBatch.json" tofile="${stage.opt.dir}/artifacts/cdap-etl-lib-${project.version}-batch.json" />
```

Comment:
lets rename the source to something closer to what it will finally be, otherwise its hard to tell just by looking which file is which.  ETLBatch -> cdap-etl-lib-batch.json